assets/js/plugins/jquery.flot.orderBars.js

```diff
@@ -88,8 +88,8 @@
         function getAxeMinMaxValues(series,AxeIdx){
             var minMaxValues = new Array();
             for(var i = 0; i < series.length; i++){
-                minMaxValues[0] = series[i].data[0][AxeIdx];
-                minMaxValues[1] = series[i].data[series[i].data.length - 1][AxeIdx];
+                minMaxValues[0] = (series[i].data[0]) ? series[i].data[0][AxeIdx]: null;
```

Comment:
@DevinWalker This was updated by gulp and it will be good if we keep them in their original version. Pr updated.